examples/llm_compression/torch/qat_with_lora/requirements.txt

```diff
@@ -0,0 +1,7 @@
+tqdm
+tensorboard
+whowhatbench @ git+https://github.com/openvinotoolkit/openvino.genai#subdirectory=tools/who_what_benchmark
+numpy>=1.23.5,<2
+# openvino==2025.1
```

Comment:
Should this be removed?
```suggestion
```

Reply:
Now it works with current openvino and WWB.